jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -417,6 +417,33 @@ async def test_mount_datums(pachyderm_resources, http_client: AsyncClient):
     assert r.json()["idx"] == 0
     assert r.json()["all_datums_received"] is True
 
+    # Test mounting a new datum while one is already mounted
```

Comment:
It's not really clear to me how this is testing the line of code you added. Shouldn't you be checking somewhere that the previous datum mounting resources are gone?

Reply:
we are testing this by verifying that the number of datums is 1 and that viewing the datum yields the expected content after mounting the new input